src/components/tx/security/redefine/RedefineScanResult/RedefineScanResult.tsx

```diff
@@ -0,0 +1,72 @@
+import { useContext } from 'react'
+import { SecurityHint, SecurityWarning } from '../../shared/SecurityWarnings'
+import { TransactionSecurityContext } from '../../TransactionSecurityContext'
+import { SecuritySeverity } from '@/services/security/modules/types'
+import { groupBy } from 'lodash'
+import { Box } from '@mui/material'
+import ExternalLink from '@/components/common/ExternalLink'
+import { FEATURES } from '@/utils/chains'
+import { useHasFeature } from '@/hooks/useChains'
+
+const MAX_SHOWN_WARNINGS = 3
+
+const REDEFINE_SIMULATION_URL = 'https://app.redefine.net/defirewall-tx/'
+
+export const RedefineScanResult = () => {
+  /* Hooks */
+  const {
+    warnings,
+    verdict,
+    isLoading,
+    error,
+    simulationUuid,
+    needsRiskConfirmation,
+    isRiskConfirmed,
+    setIsRiskConfirmed,
+  } = useContext(TransactionSecurityContext)
+  const isFeatureEnabled = useHasFeature(FEATURES.RISK_MITIGATION)
+
+  /* Evaluate security warnings */
+  const relevantWarnings = warnings.filter((warning) => warning.severity !== SecuritySeverity.NONE)
+  const shownWarnings =
+    relevantWarnings && relevantWarnings.length > MAX_SHOWN_WARNINGS
```

Comment:
According to the type `relevantWarnings` is always defined. Is it just an empty array in case there are no relevantWarnings? If so we can remove this check here.

Reply:
Also I am not sure if we need this condition at all. Can we not always just slice it? It shouldn't throw any errors even if there are fewer elements than what we slice off